Rasa_Bot/data/rules.yml

```diff
@@ -11,4 +11,17 @@ rules:
   steps: 
   - intent: who_are_you
   - action: utter_agent_name
-  - action: utter_agent_purpose
\ No newline at end of file
+  - action: utter_agent_purpose
+  
+- rule:  when user says hi at start of conversation, say hi back and ask for mood
+  conversation_start: true # only applies at a conversation start
+  steps: 
+  - intent: greeting
+  - action: action_get_name_from_database
+  - action: utter_greet
+  - action: utter_ask_user_mood
+  
+- rule: Ask the user to rephrase whenever they send a message with low NLU confidence
+  steps:
+  - intent: nlu_fallback
+  - action: utter_default
```

Comment:
Am I correct that this behavior was already in the previous version? But now it is more explicit :)

Reply:
There are 2 different types of fallback, one for not understanding the intent, and one for not knowing how to act next. We already had the one for not knowing how to act next, but did not specify the one for not understanding the intent. Of course, one might imply the other, but now we can separately tune these cases. For now, I have the same utterance for both of those cases.